### DISCUSSION TITLE
Copy settings from one tablet to another

### DISCUSSION THREAD

**Discussion by ecard0** [thanksmister/android-mqtt-alarm-panel#195] [Q&A] [answered]
Is it possible to copy the settings to a 2nd tablet?  

I'm installing a second tablet (second entrance) and I wish I could just copy the settings from the original to the new one.  

Thanks,
Eric

**Comment by thanksmister** [comment, marked as answer]
No there is no way to copy settings from one device to another.

**Reply by ecard0** [reply]
Thanks for the quick answer 👍
